frontend/src/bundles/home/components/video-section/video-section.tsx

```diff
@@ -4,34 +4,51 @@ import {
     Flex,
     Heading,
     SimpleGrid,
+    Text,
 } from '~/bundles/common/components/components.js';
+import { type VideoGetAllItemResponseDto } from '~/bundles/home/types/types.js';
 
 import { VideoCard } from '../components.js';
 
-const VideoSection: React.FC = () => {
+type Properties = {
+    videos: Array<VideoGetAllItemResponseDto> | [];
+    title: string;
+};
+
+const VideoSection: React.FC<Properties> = ({ videos, title }) => {
+    const hasVideos = videos.length > 0;
+
     return (
         <Box padding="17px 28px">
             <Flex alignItems="center" marginBottom="9px">
                 <Heading color="typography.900" variant="H3" marginRight="11px">
-                    Videos
+                    {title}
                 </Heading>
                 <Badge
                     color="background.600"
                     bg="#D1D4DB"
                     fontWeight="400"
                     padding="2px 10px"
                 >
-                    23
+                    {videos.length}
                 </Badge>
             </Flex>
-            <SimpleGrid minChildWidth="253px" spacing="20px">
-                {/* TODO: Update this mocked data */}
-                <VideoCard />
-                <VideoCard />
-                <VideoCard />
-                <VideoCard />
-                <VideoCard />
-            </SimpleGrid>
+
+            {hasVideos ? (
```

Comment:
```suggestion
            {videos.length > 0 ? (
```
I guess we can use this condition directly without creating a separate variable